src/System.Management.Automation/engine/MergedCommandParameterMetadata.cs

```diff
@@ -500,47 +500,27 @@ internal string GetParameterSetName(uint parameterSet)
 
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-linux`
TabCompletion.Miscellaneous completion tests.Test completion with exact match
```powershell
Expected a collection with size 3, but got collection with size 4 @(System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult, System.Management.Automation.CompletionResult).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1: line 909
909:             $res.CompletionMatches | Should -HaveCount 3
```